solidity/test/Coordinator_test.js

```diff
@@ -350,5 +345,173 @@ contract('Coordinator', () => {
         })
       })
     })
+
+    context("when aggregating answers", () => {
+      let oracle1, oracle2, oracle3, request, strangerOracle
+
+      beforeEach(async () => {
+        strangerOracle = h.stranger
+        oracle1 = h.oracleNode1
+        oracle2 = h.oracleNode2
+        oracle3 = h.oracleNode3
+
+        agreement = await h.newServiceAgreement({oracles: [oracle1, oracle2, oracle3]})
+        let tx = await h.initiateServiceAgreement(coordinator, agreement)
+        assert.equal(tx.logs[0].args.said, agreement.id)
+
+        mock = await h.deploy('examples/GetterSetter.sol')
+        const fHash = h.functionSelector('requestedUint256(bytes32,uint256)')
+
+        const payload = h.executeServiceAgreementBytes(agreement.id, mock.address, fHash, 1, '')
+        tx = await link.transferAndCall(
+          coordinator.address, agreement.payment, payload)
+        request = h.decodeRunRequest(tx.receipt.logs[2])
+      })
+
+      it('does not set the value with only one oracle', async () => {
+        await coordinator.fulfillOracleRequest(request.id, h.toHex(17),
+          { from: oracle1 })
+
+        const currentValue = await mock.getUint256.call()
+        assertBigNum(h.bigNum(0), currentValue)
```

Comment:
Using `0` as null here is going to cause problems for cases where 0 is a legitimate answer. Might want a method on the contract for this, instead.

Reply:
Updated by checking for the logs length instead of the consumer's value. When an oracle responds, and there are still more responses expected, no log is emitted. When aggregation occurs and the consuming contract is called, the ChainlinkFulfilled log is emitted.